lib/screens/v2/widget_instance/outside_elevator_closures.ex

```diff
@@ -0,0 +1,58 @@
+defmodule Screens.V2.WidgetInstance.OutsideElevatorClosures do
+  @moduledoc false
+
+  alias Screens.Stops.Stop
+  alias Screens.V2.WidgetInstance.CurrentElevatorClosed.Closure
```

Comment:
Since this is used from both "sibling" widget modules, defining it as a sub-module of one of them seems arbitrary. But then I also hesitate to put it anywhere else in the `WidgetInstance` namespace since it is not, itself, a Widget™. Maybe `v2/widget_instance/elevator/closure` would be okay?